src/backend/api/meals.js

```diff
@@ -1,12 +1,79 @@
 const express = require("express");
+const { request, response } = require("../app");
 const router = express.Router();
 const knex = require("../database");
 
+// maxPrice
 router.get("/", async (request, response) => {
+  let meals = knex("meal");
+  if ("maxPrice" in request.query) {
+    if (!"maxPrice") {
```

Comment:
Currently the `if` will never match:

```
> !"maxPrice"
false
```

Was this supposed to be
```suggestion
    if (!maxPrice) {
```
?